src/bin/console/pagination.cpp

```diff
@@ -105,6 +106,37 @@ void Pagination::display() {
         break;
       }
 
+      case KEY_RIGHT: {
+        ++current_column;
+        reprint = true;
+        break;
+      }
+
+      case KEY_LEFT: {
+        if (current_column > 0) {
+          --current_column;
+          reprint = true;
+        }
+        break;
+      }
+
+      case 'a': {
+        if (current_column != 0) {
+          current_column = 0;
+          reprint = true;
+        }
+        break;
+      }
+
+      case 'e': {
+        const auto last_column = _lines[current_line].length() - _size_x + 1;
+        if (_lines[current_line].length() > _size_x && current_column < last_column) {
+          current_column = last_column;
+          reprint = true;
+        }
+        break;
```

Comment:
Using `_lines[current_line].length` is problematic, as the output starts with `=== Columns`, which always fits on the screen. As a result, `[e]` does not work when pressed immediately after a print statement.

Reply:
this is how less is doing it. i guess the idea behind is that you dont end up in empty space far to the right in case you have a very long line outside the visible area.
since this can't really happen with our data, i changed it to jump to the global maximum line length.